### ISSUE TITLE
Accessibility on Desktop

### ISSUE THREAD

**Issue by c-machado** [flutter/website#1544]
Homepage -> Link options to "Twitter", "Youtube" and "Github" are being announced as "link", could you please add the name as well?.
<img width="1587" alt="screen shot 2018-10-28 at 6 58 18 pm" src="https://user-images.githubusercontent.com/33238142/47623923-885d1380-dae4-11e8-9ebf-e6a8fb4ae3de.png">

Homepage -> All images should have alt text, I wonder if it's possible to add it to the ones in homepage sections. 
<img width="119" alt="screen shot 2018-10-28 at 6 58 52 pm" src="https://user-images.githubusercontent.com/33238142/47623936-a88cd280-dae4-11e8-862a-c7693328e04a.png">

'Docs' Page -> Lists are not expandable using the keyboard.
<img width="1580" alt="screen shot 2018-10-28 at 6 59 13 pm" src="https://user-images.githubusercontent.com/33238142/47623944-be01fc80-dae4-11e8-9737-9450552c2ec4.png">

'Docs' / 'Community' Pages -> Tabs are not navigable using the keyboard. 
e.g. Navigate to: 'Get Started' -> '2. Set up an editor' -> 'Android Studio / IntelliJ', after selects 'Visual Studio Code' tab, do enter key is not showing the content for that tab.
<img width="1345" alt="screen shot 2018-10-28 at 7 00 06 pm" src="https://user-images.githubusercontent.com/33238142/47623960-f275b880-dae4-11e8-8ffe-ea00cea41ae1.png">

'Showcase' Page -> Include 'alt text' for all logo images. 
<img width="1084" alt="screen shot 2018-10-28 at 7 00 31 pm" src="https://user-images.githubusercontent.com/33238142/47623966-fd304d80-dae4-11e8-9544-3362737a9ccb.png">
